packages/v4-sdk/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@pancakeswap/v4-sdk",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "0.1.0",
+  "description": "⚒️ An SDK for building applications on top of Pancakeswap V4",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "pancakeswap",
+    "bsc"
+  ],
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "module": "dist/index.mjs",
+  "scripts": {
+    "build": "tsup",
+    "dev": "tsup --watch",
+    "test": "vitest --run",
+    "update:snapshot": "vitest -u",
+    "coverage": "vitest run --coverage",
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist"
+  },
+  "dependencies": {
+    "@pancakeswap/sdk": "workspace:*",
```

Comment:
For v4 I think we only relies on v3 right? Why do we need v2 sdk here?

Reply:
moved to devDeps